README.md

```diff
@@ -62,13 +81,70 @@ Next, configure the LSP connector for the `langserver-go` binary. To change your
                 "GOPATH": "",
             }
         }
+}
+```
+
+Finally, restart Sublime Text to start using the plugin. You may want to [disable Sublime's native tooltips](https://github.com/sourcegraph/lsp-sublime#remove-sublime-text-3-tooltips-and-goto-menu-items), as they are duplicative and interfere with this connector's tooltips.  
+
+### TypeScript/JavaScript installation
+
+First, you need to add TypeScript Sublime support the following way:
+
+```shell
+git clone git clone https://github.com/Microsoft/TypeScript-TmLanguage ~/Library/Application\ Support/Sublime\ Text\ 3/Packages/TypeScript-TmLanguage
+```
+
+Linux:
+
+```shell
+git clone https://github.com/Microsoft/TypeScript-TmLanguage ~/.config/sublime-text-3/Packages/TypeScript-TmLanguage
+```
+
+Windows:
+
+```bat
+cd "%APPDATA%\Sublime Text 3\Packages"
+git clone https://github.com/Microsoft/TypeScript-TmLanguage
+```
+
+Then install the TypeScript/JavaScript LSP server the following way:
+
+```shell
+export JSTS_DIR=...
+git clone https://github.com/sourcegraph/javascript-typescript-langserver $JSTS_DIR
+cd $JSTS_DIR
+npm install
+node_modules/.bin/tsc
+```
+
+Please make sure that `$JSTS_DIR/bin` is in `$PATH`
+
+Next, register TypeScript/JavaScript LSP client. To change your Sourcegraph settings, open `SublimeLsp.sublime-settings` by clicking `Sublime Text > Preferences > Package Settings > Sublime Lsp Connector > Settings - User`.
+Find there `clients` section, if it does not exist, create it the following way
+
+```
+{
+...
+    "clients": [
     ]
-    ...
+...
+}
+```
+
+Then, add the following client descriptor into `clients` section
+
+```
+{
+        {
+            "binary": "javascript-typescript-sublime",
```

Comment:
Do we need to call it -sublime, or is it actually just a standard-in standard-out version? "javascript-typescript-stdio" or something?